Effects/LCAS.hlsl

```diff
@@ -58,10 +57,10 @@ float4 Pass1(float2 pos) {
 	//    w  
 	//  w 1 w
 	//    w   
-	// If is not edge
-	if ((0.72f * sum) <= 1)
-		return float4(((((b + d) + (f + h)) * wRGB + e) / (1.0 + 4.0 * wRGB)).rgb, 1);
-	else
-		return float4(((((b + d) + (f + h)) * wRGB + e * (1 + sharpness) * 0.72f * sum) / (1.0 + 4.0 * wRGB)).rgb, 1);
 	// If is edge
+	if(edge >= 0.2)

```

Comment:
How about letting the user decide the threshold?

Reply:
Thats a good idea sir